### DISCUSSION TITLE
Can you give me some information about the Trust app development team?

### DISCUSSION THREAD

**Discussion by ghost** [trustwallet/wallet-core#2049] [Q&A]
Can you give me some information about the Trust app development team? I'm using this lib to build a mobile app, but I'm having a bit of trouble with the tech I want to exchange @hewigovens

**Comment by optout21** [comment]
Use Trust Developers telegram channel:  https://t.me/trust_developers

**Comment by anabelyo** [comment]
I also was searching for developers who can create an app for my business. And I found this [mobile app development](https://www.intellectsoft.net/services/mobile-app-development-services) that can build a custom iOS and Android apps development for my bakery. So we started not so long ago and I hope soon will share my app. Anyways, I expect it will help to improve my business.
